src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+function Modal(clickElm, modalParent, options) {
```

Comment:
so with webpack we don't need to wrap this whole file in a protective closure like ``(function(){...})()`` ?